### ISSUE TITLE
Can't build on latest Fedora 33

### ISSUE THREAD

**Issue by unsubtleguy** [OJ/gobuster#282]
tl;dr - If on Fedora 33, use the release instead, your build of golang is too old
----

Hey, opening and self closing a ticket in the hopes it saves OJ from doing it :)

I've got a CI job pulling in Gobuster on Fedora 33, which has been failing since about 3 days ago, coinciding with the changes in https://github.com/OJ/gobuster/commit/2f636e758cbc131732b91a492d06065e28745899, which summarised:

* CI uses 1.16.0 instead of 1.15.8
* Remove io/ioutil
* Use io instead
* A bunch of other stuff, but none relevant for the break

When trying to build on Fedora 33 (i.e. latest), one would get the following error:
```
$ go get -u github.com/OJ/gobuster
# github.com/OJ/gobuster/libgobuster
go/src/github.com/OJ/gobuster/libgobuster/http.go:114:15: undefined: io.ReadAll
go/src/github.com/OJ/gobuster/libgobuster/http.go:122:25: undefined: io.Discard
```

Fedora 33 is using an old version of Golang, so this is hardly surprising.

```
dnf list golang
Installed Packages
golang.x86_64   1.15.8-1.fc33 
```

I've documented it at https://github.com/pentesterwtf/ansible-desktop/issues/15, I think the solution ends up being to either pull in latest Golang or just use the release of Gobuster.

Thanks for the tooling!

**Comment by OJ** [comment]
Hey there. Thanks for this. Did you close this because it just requires latest Golang or was there some other solution?

**Comment by unsubtleguy** [comment]
Hey OJ,

It's a funny one - Fedora 33 looks like it's not going to swap to golang 1.16 anytime soon, but fedora 34 is - so I've just swapped back to using a release for gobuster.

Given fedora 34 is out in March (at time of writing, this month!), I'm probably going to just use the release until I upgrade.

Thanks for your work!

**Comment by OJ** [comment]
@unsubtleguy ok great, that's good to know. Thanks for the info, much appreciated! 😃
